AISKU/snippet/snippet.js

```diff
@@ -1,106 +1,304 @@
-﻿var sdkInstance = "appInsightsSDK";
-window[sdkInstance] = "appInsights";
-var aiName = window[sdkInstance]; // provide non default instance name through key appInsightsSDK
-var aisdk = window[aiName] || (function (aiConfig) {
-    var appInsights = {
-        config: aiConfig
-    };
-    appInsights.initialize = true; // initialize sdk on download
-
-    // Assigning these to local variables allows them to be minified to save space:
-    var localDocument = document, localWindow = window, scriptText = "script";
-    setTimeout(function () {
-        var scriptElement = localDocument.createElement(scriptText);
-        scriptElement.src = aiConfig.url || "https://az416426.vo.msecnd.net/scripts/b/ai.2.min.js";
```

Comment:
I removed the aiConfig.url and replaced it with the snippetConfig.src.

I assume this is ok as users will be updating to this new snippet and the url is not used elsewhere.

Reply:
After thinking about this for a while, I've re-added this so we don't break anyone.